bundles/io/org.openhab.io.dropbox/src/main/java/org/openhab/io/dropbox/internal/DropboxSynchronizer.java

```diff
@@ -137,28 +148,31 @@
             "/configurations.*");
 
     /**
-     * defines a comma separated list of regular expressions which matches the filenames to upload to Dropbox (optional;
-     * defaults to '/configurations.*, /logs/.*, /etc/.*')
+     * Defines a comma separated list of regular expressions which matches the
+     * filenames to upload to Dropbox (optional; defaults to '/configurations.*,
+     * /logs/.*, /etc/.*')
      */
     private static List<String> uploadFilterElements = DEFAULT_UPLOAD_FILE_FILTER;
 
     /**
-     * defines a comma separated list of regular expressions which matches the filenames to download from Dropbox
-     * (optional; defaults to '/configurations.*')
+     * Defines a comma separated list of regular expressions which matches the
+     * filenames to download from Dropbox (optional; defaults to '/configurations.*')
      */
     private static List<String> downloadFilterElements = DEFAULT_DOWNLOAD_FILE_FILTER;
 
     /**
-     * operates the Synchronizer in fake mode which avoids sending files to or from Dropbox. This is meant
-     * as a test mode for the filter settings. (optional; defaults to false)
+     * Operates the Synchronizer in fake mode which avoids sending files to or
+     * from Dropbox. This is meant as a test mode for the filter settings.
+     * (optional; defaults to false)
      */
     private static boolean fakeMode = false;
 
     private static boolean isProperlyConfigured = false;
 
     private static DropboxSynchronizer instance = null;
 
-    private static final DbxAppInfo appInfo = new DbxAppInfo(DropboxSynchronizer.appKey, DropboxSynchronizer.appSecret);
+    private static final DbxAppInfo appInfo =
+        new DbxAppInfo(DropboxSynchronizer.appKey, DropboxSynchronizer.appSecret);
```

Comment:
I agree. I had considered reworking the entire binding to make all the static items non-static, too.

I'll work those changes up and add them to this PR.